x-pack/plugins/upgrade_assistant/public/application/components/es_deprecations/deprecation_tab_content.tsx

```diff
@@ -28,6 +34,61 @@ export interface CheckupTabProps extends UpgradeAssistantTabProps {
   checkupLabel: string;
 }
 
+export const filterDeps = (level: LevelFilterOption, search: string = '') => {
```

Comment:
👍 This was existing behavior. I've changed it to a `union`.  There are several other places in the code where enums are still being used, but I think this can be addressed on a case-by-case basis.